README.md

```diff
@@ -477,6 +488,11 @@ urlpatterns = [
 
 ### 6.0.0
 
+* Drop Django 1.11, 2.0, and 2.1, and Python 2,7, 3.4, and 3.5 support
+* Add Django 2.2 and 3.0, and Python 3.6, 3.7, and 3.8 support
```

Comment:
I have a request, and I hope it's not unreasonable of me.

Please don't drop 1.11 support until the release after you have introduced 2.2 support. by having a versions that supports two LTS versions at once, you greatly ease the life of people on an upgrade path.

Reply:
I'm not sure how to approach that. The guidance to me from the person above me is to only include in a release versions of Python/Django that are officially supported. The reason why I dropped 1.11 is because the EoL is April. There are over 20 apps that were included in the last Pinax release. A release is a massive task and there is very little contribution to make it happen. It may still be in process by the EoL date in April and it's not likely there will be another one soon after. This app is also a problem app and I don't think can be released anyway, because the tests do not pass at this time.

Reply:
Fair enough, I don't want to derail your release process, or add undue pressure.